### PR TITLE
Expose different ring states in the metrics

### PR DESCRIPTION
**Pull request by tomwilkie**
Now the graph will have Active, Leaving and unhealthy in it.

### CODE REVIEW

**Comment by tomwilkie** [comment]
Will need post-merge review #97

File: ring/ring.go [hunk 1]
```diff
@@ -14,8 +14,7 @@ import (
 )
 
 const (
-	healthyLabel   = "healthy"
-	unhealthyLabel = "unhealthy"
+	unhealthy = "Unhealthy"
```
Comment (jml) [inline ring/ring.go]: Why not make `unhealthy` an `IngesterState`?

**Comment by jml** [comment]
LGTM.

File: ring/ring.go (hunk 1 above)
Comment (jml) [inline ring/ring.go]: I went to make this change and then saw your point. Maybe if Go had subtypes.

**Comment by tomwilkie** [comment]
Maybe if go had a type system?

**Pull request merged**